python/cudf/cudf/tests/test_groupby.py

```diff
@@ -789,24 +799,35 @@ def test_groupby_index_type():
 
 
 @pytest.mark.parametrize(
-    "interpolation", ["linear", "lower", "higher", "midpoint", "nearest"]
+    "interpolation", ["linear", "lower", "higher", "nearest", "midpoint"]
 )
-def test_groupby_quantile(interpolation):
+@pytest.mark.parametrize("q", [0.25, 0.4, 0.5, 0.7, 1])
+def test_groupby_quantile(interpolation, q):
     raw_data = {
-        "x": [1, 2, 3, 1, 2, 2, 1, None, 3, 2],
         "y": [None, 1, 2, 3, 4, None, 6, 7, 8, 9],
+        "x": [1, 2, 3, 1, 2, 2, 1, None, 3, 2],
     }
-    pdf = pd.DataFrame(raw_data)
+    # Pandas>0.25 now casts NaN in quantile operations as a float64
+    # # so we are filling with zeros.
+    pdf = pd.DataFrame(raw_data).fillna(0)
```

Comment:
Should we be testing this across multiple datatypes, then?

Reply:
Good point. Yes we should.